Lib/typing.py

```diff
@@ -1782,14 +1782,22 @@ def __init__(cls, *args, **kwargs):
             )
 
     def __subclasscheck__(cls, other):
+        if not isinstance(other, type):
```

Comment:
I'm not sure we even need this check, `_abc_subclasscheck` does this already.

Reply:
This could make a difference in cases involving multiple inheritance of the metaclass, I suppose. That might not be a case we need to care about as this metaclass is private and shouldn't be used directly.

Reply:
Thanks, that makes sense.